zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServerBean.java

```diff
@@ -403,4 +403,10 @@ public int getLargeRequestThreshold() {
     public void setLargeRequestThreshold(int threshold) {
         zks.setLargeRequestThreshold(threshold);
     }
+
+    ///////////////////////////////////////////////////////////////////////////
```

Comment:
why need this?

Reply:
I have added this as a seperator for this function. Since the other functions added above this function had used this as a seperator, I followed the same inorder to maintain consistency

Reply:
I have removed this line as requested :)